deploy/environment.sh.example

```diff
@@ -1,29 +1,17 @@
 #! /bin/bash
 
-# Replace "your-entity-name" on LINE 16 with the name of your public W&B model registry entity name
-#  
-# This script sets up the WANDB_ENTITY environment variable necessary to 
-# download models from a public Weights & Biases model registry. 
+# Replace "your-deploy-key-here" on LINE 19 with your deploy key
 # 
 # For use, this script should be sourced into your current shell instance with `source deploy/environment.sh`
 # or `./deploy/environment.sh` - the environment variable will be dropped after the shell session terminates.
 
 
-# wandb entity name
-if [ -n "$WANDB_ENTITY" ]
+# Docker Hub deploy key
+if [ -n "$DOCKER_HUB_DEPLOY_KEY" ]
```

Comment:
This check isn't needed. If you set `DOCKER_HUB_DEPLOY_KEY` twice, it will simply contain the second value. There is no need to check if the variable already exists, we are setting it to what we want in this file.